wasm/build.ts

```diff
@@ -38,12 +38,14 @@ async function shell(program: string, args: string[]) {
 
   await new Promise<void>((resolve, reject) => {
     child.on('exit', code => {
-      if (code !== 0) {
-        reject(new Error(
-          `Failed shell command (code=${code}): ${[program, ...args].join(' ')}`
-        ));
-      } else {
+      if (code === 0) {
         resolve();
```

Comment:
This is the only part where the code has been changed, according to the ESLint rules. Same logic tho.